setup.py

```diff
@@ -279,10 +279,10 @@ def run(self):
 
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
-    'ipython>=4.0.0dev0',
-    'ipykernel',
+    'ipython>=4.0.0',
+    'ipykernel>=4.2.0',
     'traitlets',
-    'notebook',
+    'notebook>=4.1.0b1',
```

Comment:
I don't think so. If you look just above, we had `ipython>=4.0.0dev0` for a long time.